cli/azd/pkg/azdo/azdo.go

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package azdo
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/microsoft/azure-devops-go-api/azuredevops"
+)
+
+var (
```

Comment:
nit: Should be `const` ?
```suggestion
const (
```

Reply:
We specifically made it a var so that we can override values at a later time. For example AzDo host name to support Azure DevOps server (and not just hosted SaaS.)